code-of-conduct_events

```diff
@@ -0,0 +1,104 @@
+
+3. Wider community response to Incident:
+
+    If incident results in corrective action, the community should support
+    the decision made by the Help in Step 2 if they choose corrective action,
+    like ending a talk early or banning from the event and other IPFS 
+    communication channels, as well as support those harmed by the incident, 
+    either publicly or privately (whatever individuals are comfortable with).
+
+    If the Help in Step 2 run into issues implementing the CoC, then the
+    Help should come to the community with these issues and the community
+    should revise the CoC as they see fit.
+
+    In Real Life people will have opinions about how the CoC is enforced.
+    People will argue that a particular decision was unfair, and others will
+    say that it didn't go far enough. We can't stop people having
+    opinions, but what we could do here is have constructive discussions
+    that lead to something tangible (affirmation of decision, change in CoC,
+    modify decision, etc).
+
+### Sanctions
+
+Participants asked to stop any harassing behavior are expected to comply
+immediately. If a participant engages in harassing behavior, organizers may
+take any action they deem appropriate, including warning the offender,
+expulsion from the IPFS event, or banning the offender from a chatroom, mailing list, or GitHub organization.
+
+Specific sanctions may include but are not limited to:
+
+* warning the harasser to cease their behavior and that any further reports
+will result in other sanctions
+* requiring that the harasser avoid any interaction with, and physical
+proximity to, their victim for the remainder of the event
+* early termination of a talk that violates the policy
+* not publishing the video or slides of a talk that violated the policy
+* not allowing a speaker who violated the policy to give (further) talks at
+the event
+* immediately ending any event volunteer responsibilities and privileges the
+harasser holds requiring that the harasser not volunteer for future IPFS
+events (either indefinitely or for a certain time period)
+* requiring that the harasser immediately leave the event and not return
+* banning the harasser from future events (either indefinitely or for a
+certain time period)
+* publishing an account of the harassment
+
+IPFS event organizers will help participants contact hotel/venue security or local law enforcement,
+provide escorts, or otherwise assist those experiencing harassment to feel
+safe for the duration of the event.
+
+In the case of organized events, those responsible for the event should have readily available the following information:
+
+* **Conference organizer:** [ORGANIZER NAME], [PHONE NUMBER]
+* **[CONFERENCE HOTEL NAME]:** [PHONE NUMBER]
+* **[MUNICIPALITY] Police Department**: [PHONE NUMBER]
+* **[LOCAL SEXUAL ASSAULT/CRISIS LINE]:** [PHONE NUMBER]
+* **[TAXI COMPANY NAME]:** [PHONE NUMBER]
```

Comment:
Seems like this should be part of an internal protocol for the event organization (which may vary from place to place). I would remove this as it does not pertain to specifying acceptable conduct in an event.